src/components/Translate.tsx

```diff
@@ -1,14 +1,24 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
-interface ITranslate {
+export interface ITranslate {
```

Comment:
I would move this interface into `api.tsx` file, since it's the data of the api and it's more natural place to be located for now.